bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookKeeperServerStats.java

```diff
@@ -46,6 +46,7 @@ public interface BookKeeperServerStats {
     // Stats
     String ADD_ENTRY_REQUEST = "ADD_ENTRY_REQUEST";
     String ADD_ENTRY = "ADD_ENTRY";
+    String WRITE_THREAD_QUEUE_LATENCY = "WRITE_THREAD_QUEUE_LATENCY";
```

Comment:
use `WRITE_THREAD_QUEUED_LATENCY` might be better.

Reply:
Fixed,PTAL,thanks!